python/delta_sharing/converter.py

```diff
@@ -14,44 +14,86 @@
 # limitations under the License.
 #
 from decimal import Decimal
-from json import loads
 from typing import Any, Callable, Dict
 
 import numpy as np
 import pandas as pd
 
 
-def to_converters(schema_string: str) -> Dict[str, Callable[[str], Any]]:
-    schema_json = loads(schema_string)
+def _get_dummy_column(schema_type):
+    if schema_type == "boolean":
+        return pd.Series([False])
+    elif schema_type == "byte":
+        return pd.Series([0], dtype="int8")
+    elif schema_type == "short":
+        return pd.Series([0], dtype="int16")
+    elif schema_type == "integer":
+        return pd.Series([0], dtype="int32")
+    elif schema_type == "long":
+        return pd.Series([0], dtype="int64")
+    elif schema_type == "float":
+        return pd.Series([0], dtype="float32")
+    elif schema_type == "double":
+        return pd.Series([0], dtype="float64")
+    elif isinstance(schema_type, str) and schema_type.startswith("decimal"):
+        return pd.Series([0], dtype=np.dtype("O"))
+    elif schema_type == "string":
+        return pd.Series([0], dtype=np.dtype("O"))
+    elif schema_type == "date":
+        return pd.Series([pd.Timestamp(0).date()])
+    elif schema_type == "timestamp":
+        return pd.Series([pd.Timestamp(0).date()], dtype=np.dtype("datetime64[ns]"))
+    elif schema_type == "binary":
+        return pd.Series([0], dtype=np.dtype("O"))
+    elif isinstance(schema_type, dict) and schema_type["type"] in ("array", "struct", "map"):
+        return pd.Series([0], dtype=np.dtype("O"))
+
+    raise ValueError(f"Could not parse datatype: {schema_type}")
+
+
+def get_empty_table(schema_json: dict) -> pd.DataFrame:
+    assert schema_json["type"] == "struct"
+
+    dummy_table = pd.DataFrame(
+        {field["name"]: _get_dummy_column(field["type"]) for field in schema_json["fields"]}
+    )
+    return dummy_table.iloc[0:0]
+
+
+def to_converters(schema_json: dict) -> Dict[str, Callable[[str], Any]]:
```

Comment:
nit: could you add a comment to explain what this does and the meaning of the return value.